mmr-lib/Randomizer.cs

```diff
@@ -1544,22 +1574,22 @@ public RandomizedResult Randomize(BackgroundWorker worker, DoWorkEventArgs e)
                     worker.ReportProgress(35, "Making gossip quotes...");
 
                     //gossip
-                    SeedRNG();
```

Comment:
It's important to re-seed the RNG before different sections so that the various random options don't affect each other. If you take these out, you'll get different background music with the same seed if you randomize different items.

Reply:
Opted to remove reseeding the RNG for Gossip Hints, since it's output is dependent on the item shuffle anyway. Added the others back in